crypto/ui/ui_openssl.c

```diff
@@ -503,7 +503,6 @@ static int echo_console(UI *ui)
 {
 # if defined(TTY_set) && !defined(OPENSSL_SYS_VMS)
     memcpy(&(tty_new), &(tty_orig), sizeof(tty_orig));
-    tty_new.TTY_FLAGS |= ECHO;
 # endif
 
 # if defined(TTY_set) && !defined(OPENSSL_SYS_VMS)
```

Comment:
Is it a no on removing these two pre-processor directives? I mean both #ifs are equivalent, so it's readability issue, which becomes even more acute with memcpy alone in #if-#endif...

Reply:
> Is it a no on removing these two pre-processor directives?

Nah, just me forgetting.  Will fix, thanks for the reminder.